lib/components/narrative/default/default-itinerary.js

```diff
@@ -6,6 +6,7 @@ import ItineraryBody from '../line-itin/connected-itinerary-body'
 import ItinerarySummary from './itinerary-summary'
 import SimpleRealtimeAnnotation from '../simple-realtime-annotation'
 import { getTotalFareAsString } from '../../../util/state'
+import { setActiveLeg } from "../../../actions/narrative";
```

Comment:
@robertgregg3, looks like the linter flagged some issues here. Just a heads up that for this repo we use single quotes and no semi-colons. Also, this looks to be an unused import, so it can just be deleted.

```suggestion
```

Reply:
Thanks Landon. All sorted.